core/tee/tee_svc.c

```diff
@@ -716,7 +716,7 @@ static TEE_Result tee_svc_update_out_param(
 	bool have_private_mem_map = is_user_ta_ctx(called_sess->ctx);
 
 	for (n = 0; n < TEE_NUM_PARAMS; n++) {
-		switch (TEE_PARAM_TYPE_GET(param->types, n)) {
```

Comment:
ok, `param` here is already a temporary copy of the param structure provided to called TA and called TA cannot change the its "type" field.
Closing this useless PR.